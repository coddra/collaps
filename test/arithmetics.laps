1 + 2 print
3 - 4 print
5 * 6 print
8 / 7 print
10 % 9 print
