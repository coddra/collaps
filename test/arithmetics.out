3
-1
30
1
1
